handler/handler.go

```diff
@@ -1041,6 +1044,77 @@ func (h *Handler) doExecWatch(ctx context.Context, w io.Writer, opt metacmd.Opti
 
 // doExecChart executes a single query against the database, displaying its output as a chart.
 func (h *Handler) doExecChart(ctx context.Context, w io.Writer, opt metacmd.Option, prefix, sqlstr string, qtyp bool, bind []interface{}) error {
+	stdout, _, _ := h.l.Stdout(), h.l.Stderr(), h.l.Interactive()
+	typ := env.TermGraphics()
+	if !typ.Available() {
+		return text.ErrGraphicsNotSupported
+	}
+	if _, ok := opt.Params["help"]; ok {
+		fmt.Fprintln(stdout, text.ChartUsage)
+		return nil
+	}
+	cfg, err := charts.ParseArgs(opt.Params)
+	if err != nil {
+		return err
+	}
+	start := time.Now()
+	// query
+	rows, err := h.DB().QueryContext(ctx, sqlstr, bind...)
+	if err != nil {
+		return err
+	}
+	// get cols
+	cols, err := drivers.Columns(h.u, rows)
+	if err != nil {
+		return err
+	}
+	// process row(s)
+	transposed := make([][]string, len(cols))
+	clen, tfmt := len(cols), env.GoTime()
+	for rows.Next() {
```

Comment:
Just to make sure, there is no need for a call to a defer rows.Close or something like that? It's a common pattern, so I'm asking

Reply:
good catch

Reply:
good catch